### PR TITLE
Enhance the Frontend & Make it more Attractive 

### PR DESCRIPTION
**Pull request by anushavc**
#### Issue Number
ISSUE #4 

#### Describe the changes you've made

I have made changes to all the HTML files and the style.css file to enhance the frontend of the project.  I have added a new svg file which I have used in the start.html.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

#### Checklist

- [x] My code follows the code style of this project.
- [x] I have performed a self-review of my own code.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by SanjayMarreddi — CHANGES_REQUESTED** [review]
@anushavc Overall It looks good. Here are few suggestions for the Home Page (`Start.html`) :-
 - There are 3 boxes right, Reduce the dimensions of the last 2 boxes & their content accordingly.
 - Also, keep all the 3 boxes align in center position of the web page. (Currently, They look aligned towards right side.)
 - It would be better if you could even add some related Images to impose interest to user's to use our project.
 
 Good Luck !!

**Comment by anushavc** [comment]
I have made the suggested changes. Do let me know if I need to make any more

File: templates/Start.html [hunk 1]
```diff
@@ -62,47 +76,105 @@
...
+
+  <!--about section-->
+  <div style="display:flex;flex-direction:column; align-items:center;
+  justify-content:center;
+  text-align:center;">
+    <div class="animation-area" style="height: 100vh;width:100%;display:flex;flex-direction:column; align-items:center;
+    justify-content:center;text-align:center;">
+    <div class="animation-area .container-sm" style="height: 80vh;width:70%;display:flex;flex-direction:column; align-items:center;
+    justify-content:center;text-align:center;background-color:#0A043C;margin:30px;color:white">
+    <p style="font-size:xx-large;" >About the project</p>
```
Comment (SanjayMarreddi) [inline templates/Start.html]: Change the line as follows 
`<p style="font-size:xx-large;" >Glimpse of the project !</p>`

File: templates/Start.html [hunk 2]
```diff
@@ -62,47 +76,105 @@
...
+    <p style="font-size:x-large;" >To detect your emotion using the webcam feed</p>
```
Comment (SanjayMarreddi) [inline templates/Start.html]: Remove the line number `111`

**Review by SanjayMarreddi — CHANGES_REQUESTED** [review]
@anushavc Everything is alright. Just look at the below 2 slight Modifications

**Review by SanjayMarreddi — APPROVED** [review]

**Pull request merged**